### DISCUSSION TITLE
How to set board defaults?

### DISCUSSION THREAD

**Discussion by vkvark** [dbuezas/lgt8fx#262] [Q&A] [answered]
Hi there.

Each time I select the lgt8fx core, I need to manually set the board parameters - at least the clock which is 32 MHz internal by default. How can I change this default setting, e.g. to External 16 MHz?

**Comment by dbuezas** [comment, marked as answer]
I don't think you can do this in the Arduino IDE. The only option I see is to edit the boards.txt file inside the core

**Reply by vkvark** [reply]
Can you tell me how to set board defaults in the boards.txt file?

**Reply by dbuezas** [reply]
Where the file is depends on your OS. Google "where is the arduino boards.txt file in windows / mac / linux".
Then go there and find the lgt8fx package and find the boards.txt file there. Then move the options you want to be default to the top of their section.

**Reply by vkvark** [reply]
Thanks a lot, this is exactly what I was looking for!

**Reply by dbuezas** [reply]
beware that if there is an update for this core and you install it, the boards.txt will be replaced with a new one and you'll have to apply the trick again :)

**Reply by vkvark** [reply]
Sure. I've made a backup of the modified boards.txt file :).

**Comment by dbuezas** [comment]
Reopening, as closing discussions makes them less visible
